sites/healthcarepackaging/index.js

```diff
@@ -21,4 +22,5 @@ module.exports = startServer({
   version,
   onStart: app => app.set('trust proxy', 'loopback, linklocal, uniquelocal'),
   onAsyncBlockError: e => newrelic.noticeError(e),
+  redirectHandler: ({ from }) => redirect.check(from),
```

Comment:
Once you export the handler as a function (and use the destructured args), you can change this:

```
redirectHandler: ({ from }) => redirect.check(from),
```

To:

```
redirectHandler,
```

This change applies to all files.